rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -504,23 +535,63 @@ private void callUpdateCollections() throws Exception {
         bridgeSupport.save();
     }
 
-    private BridgeSupport getBridgeSupportFromExecutionBlock(Block executionBlock) {
-        FederationSupport fedSupport = FederationSupportBuilder.builder()
+    private void assertPeginsShouldWorkToFed(Address federationAddress, String sender) throws Exception {
```

Comment:
The name method is different from what the method is doing. It should express precisely that it is first doing a pegin and then validating that the pegin was processed.